src/IotHub/IotHub/Models/PSIotHubInputProperties.cs

```diff
@@ -63,11 +63,6 @@ public class PSIotHubInputProperties
         [JsonProperty(PropertyName = "comments")]
         public string Comments { get; set; }
 
-        /// <summary>
-        /// </summary>
-        [JsonProperty(PropertyName = "operationsMonitoringProperties")]
-        public PSOperationsMonitoringProperties OperationsMonitoringProperties { get; set; }
```

Comment:
@anusapan this property should not be removed

Reply:
changes reverted